Ansible/.env.j2

```diff
@@ -0,0 +1,11 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=production
+
+# Change the following values for local development.
+SECRET_KEY=secret-key
+APIKEY={{ api_key }}
+APIToken={{ api_token }}
+TrelloBID={{ trellobid }}
+URL="https://api.trello.com/"
+PATH=$PATH:/etc/poetry/bin
```

Comment:
I think this last entry will be unnecessary, since this will only be loaded after we've run `dotenv` which happens when Poetry starts our app